test/cmdlineTests/optimizer_no_immutables_assignments/err

```diff
@@ -0,0 +1,13 @@
+Warning: SPDX license identifier not provided in source file. Before publishing, consider adding a comment containing "SPDX-License-Identifier: <SPDX-License>" to each source file. Use "SPDX-License-Identifier: UNLICENSED" for non-open-source code. Please see https://spdx.org for more information.
```

Comment:
What would be the right place for the test? I'm not particularly happy adding it to **cmdlineTests**.